primer/src/Primer/API.hs

```diff
@@ -829,6 +829,29 @@ viewTreeExpr e0 = case e0 of
             }
       viewFallback = case fb of
         CaseExhaustive -> Nothing
+        CaseFallback rhs ->
+          let
+            -- these IDs will not clash with any others in the tree,
+            -- since node IDs in the input expression are unique,
+            -- and don't contain non-numerical characters
+            boxId = nodeId <> "Pwild"
+            patternRootId = boxId <> "B"
+           in
+            Just $
+              Tree
+                { nodeId = boxId
+                , body =
+                    BoxBody . RecordPair Flavor.Pattern $
+                      ( Tree
+                          { nodeId = patternRootId
+                          , body = TextBody $ RecordPair Flavor.PatternWildcard (Name Nothing "_")
```

Comment:
It is somewhat odd to render the wildcard match as a text "_" node...
Any better ideas?

Reply:
We also considered not rending the pattern box at all in this case and going straight into the output. However, we realized that while this might be the nicest solution for the wildcard pattern, it wouldn't mesh well with deep patterns (once we can support them) that have nested wildcard patterns.

Reply:
We now render it as its own flavor with no body